packages/loader/README.md

```diff
@@ -0,0 +1,41 @@
+# loader
+
+purely makes use of css to provide a ripple effect.
```

Comment:
Could we provide a better description?  i.e. what does this component do?  why would anyone want to use it?  it is not very important to only describe how it works technically.  Please look at examples from some of the other packages in this repo.

Reply:
ill add a more descriptive description as suggested